docs/resource-schemas.md

```diff
@@ -321,6 +332,37 @@ have the following values:
 | `cross-platform` | Runs natively on iOS, Android, and/or desktop. |
 | `web`            | Runs on the web.                               |
 
+#### Asset docs
+
+Asset docs is used to determine which supporting documentation is available to fetch and render as
+page tabs in the asset details for the asset as well as it's location. For each optional document
+key, a path should be included; The path **must** be either a Github url according to the
+`https://{host}/{org}/{repo}/blob/{branch}/{path-to-file}` format, or a path relative to the
+carbon.yml file location.
+
+Currently supported document keys are:
+
+| docKey              | Description                                                                                                                                                                                       |
```

Comment:
I'd replace `docKey` with `Document` since `docKey` looks like it'd be used in the YML.

Reply:
done!